test/models/job_execution_test.rb

```diff
@@ -2,7 +2,7 @@
 require_relative '../test_helper'
 require 'ar_multi_threaded_transactional_tests'
 
-SingleCov.covered! uncovered: 11 # randomly says it only has 10 ... keep at 11
```

Comment:
🤔

Reply:
I'm hoping that bug is gone now ... might have to re-introduce that :D